second-gen/packages/core/components/progress-circle/ProgressCircle.base.ts

```diff
@@ -46,6 +47,24 @@ export abstract class ProgressCircleBase extends SizedMixin(SpectrumElement, {
     @property({ type: Number })
     public progress = 0;
 
+    /**
+     * Stroke width for the progress circle.
+     */
+    @property({ type: Number, reflect: false })
+    public get strokeWidth(): number {
```

Comment:
This seems a nicer API than the current but would be good to have a discussion re: handling the evolution of the APIs between S1 & S2. S1 progress circle _could_ have a new strokeWidth customization and eschew the current makeRotation function in favor of letting CSS do the calculations (sans transformation).

Reply:
I might remove this entirely. After talking with @aramos-adobe, it doesn't make sense to allow strokeWidth to vary because it corresponds to the input step and since this represents a percent out of 100, we likely only want to support single steps.